Sources/Variable.swift

```diff
@@ -123,6 +123,34 @@ public func ==(lhs: Variable, rhs: Variable) -> Bool {
   return lhs.variable == rhs.variable
 }
 
+/// A structure used to represet range of to integer values expressed as `from to to`.
+/// Values should be numbers (they will be converted to integers) and `from` should be less than `to`.
+/// Rendering this variable produces closed range `from...to`
+public struct RangeVariable: Resolvable {
+  public let from: Resolvable
+  public let to: Resolvable
+
+  public func resolve(_ context: Context) throws -> Any? {
+    let fromResolved = try from.resolve(context)
+    let toResolved = try to.resolve(context)
+
+    guard let from = fromResolved.flatMap(toNumber(value:)).flatMap(Int.init) else {
+      throw TemplateSyntaxError("from value \(fromResolved ?? "nil") is not an int")
+    }
+
+    guard let to = toResolved.flatMap(toNumber(value:)).flatMap(Int.init) else {
+      throw TemplateSyntaxError("to value \(toResolved ?? "nil") is not an int")
+    }
+
+    guard from < to else {
```

Comment:
Why should `from` be less than `to`? Nothing is stopping us from adding decreasing ranges, and add a check below to if need be, `reverse` the range.

Reply:
Because I used CountableClosedRange that requires that (or is it <= ?). We can of course switch to array here and remove this condition.

Reply:
Regarding reversing this will be hard to do using filter with such syntax: `a to b | reverse` will try to apply filter to b, not to the range. But I guess if you are constructing range manually you wouldn't need to reverse it as you can create it already in a write direction.

Reply:
No, I meant more something like:
```
let range = CountableClosedRange<Int>(uncheckedBounds: (lower: from, upper: to)).reversed()
```

Reply:
More complete bit of code:
```swift
let lower = min(from, to)
let upper = max(from, to)
var range = CountableClosedRange<Int>(uncheckedBounds: (lower: from, upper: to))
if from > to {
  range = range.reversed()
}
```